database/src/init.sql

```diff
@@ -46,4 +51,12 @@ values  ('news', 'generic news articles', '#cc0099'),
         ('food', 'food, cooking, and recipes articles', '#5039a3'),
         ('sport', 'sport, fitness, and exercise articles', '#39a375');
 
+insert into Authors (name)
+values ('John Doe'),
+        ('Jane Doe'),
+        ('Joe Blow'),
+        ('Joey Sixpack'),
+        ('Tommy Writer'),
+        ('Sarah Jones');
```

Comment:
Can we get ChatGPT to generate something like 100 believable names instead.

Reply:
```plaintext
generate 100 author names in a sql insert statement
```

```sql
insert Author (name) values
('Eleanor Mitchell'),
('Benjamin Hudson'),
('Olivia Peterson'),
('William Anderson'),
('Sophia Richardson'),
('Daniel Martin'),
('Ava Thompson'),
('James Rodriguez'),
('Emma Foster'),
('Michael Brooks'),
('Isabella Hughes'),
('John Mitchell'),
('Mia Carter'),
('David Cooper'),
('Sophie Walker'),
('Joseph Wright'),
('Amelia White'),
('Matthew King'),
('Emily Scott'),
('Christopher Morgan'),
('Grace Davis'),
('Andrew Lee'),
('Avery Evans'),
('Ryan Turner'),
('Chloe Bennett'),
('Nicholas Murphy'),
('Harper Collins'),
('Ethan Hill'),
('Liam Allen'),
('Aria Phillips'),
('Daniel Young'),
('Layla Hall'),
('Alexander Wright'),
('Charlotte Adams'),
('Logan Green'),
('Madison Cook'),
('Benjamin Gray'),
('Aubrey Nelson'),
('Samuel Parker'),
('Elizabeth Hall'),
('Jack Martinez'),
('Ella Morris'),
('William Brooks'),
('Scarlett Bell'),
('David Lewis'),
('Victoria Price'),
('Henry Sanchez'),
('Luna Edwards'),
('Owen Turner'),
('Stella Rivera'),
('Daniel Cooper'),
('Nova Mitchell'),
('Sebastian Ward'),
('Zoe Murphy'),
('Lucas Martinez'),
('Penelope Walker'),
('Caleb Rogers'),
('Lily Howard'),
('Jackson Moore'),
('Sofia Perry'),
('Gabriel Morgan'),
('Avery Foster'),
('Elijah Bennett'),
('Addison Russell'),
('Grayson Diaz'),
('Hazel Johnson'),
('Ethan Myers'),
('Aria White'),
('Carter Peterson'),
('Grace Adams'),
('Jayden Hill'),
('Riley Reed'),
('Leo Turner'),
('Ella Rogers'),
('Alexander Bell'),
('Chloe Parker'),
('Henry Hayes'),
('Scarlett Wright'),
('David Cooper'),
('Layla Evans'),
('James Allen'),
('Aurora Mitchell'),
('Michael Ward'),
('Sophia Foster'),
('Jackson Reed'),
('Ava Collins'),
('William Hernandez'),
('Emily Carter'),
('Daniel King'),
('Olivia Scott'),
('Benjamin Adams'),
('Mia Stewart'),
('Noah Hughes'),
('Emma Morgan'),
('Logan Fisher'),
('Isabella Turner'),
('Liam Mitchell'),
('Avery Lopez'),
('Eleanor Howard'),
('Lucas Richardson'),
('Charlotte Miller'),
('Owen Cooper'),
('Amelia Murphy'),
('Elijah Wilson'),
('Penelope Green'),
('Caleb Martinez'),
('Lily Bailey'),
('Jack Smith'),
('Zoe Foster'),
('Ryan Johnson'),
('Aria Gray'),
('Samuel Turner'),
('Hannah White'),
('Daniel Brooks'),
('Ava Wright'),
('Matthew Parker'),
('Sophie Reed'),
('Alexander Cooper'),
('Grace Davis'),
('William Ward'),
('Mia Rogers');
```

Reply:
nice idea, have implemented